ldaptor/protocols/pureldap.py

```diff
@@ -212,10 +212,11 @@ def toWire(self):
             ], tag=self.tag).toWire()
 
     def __repr__(self):
+        converter = to_bytes if six.PY2 else to_unicode
```

Comment:
maybe, instead of importing `to_unicode` here, we can have a `repr_converter` helper which does the `converter` job and same helper can be used in ` ldaptor/protocols/pureber.py` and in other places... this seems like a common pattern